src/plopp/graphics/canvas.py

```diff
@@ -115,14 +115,14 @@ def autoscale(self, draw=True):
     def draw(self):
         self.fig.canvas.draw_idle()
 
-    def savefig(self, filename: str = None):
+    def savefig(self, filename: str, dpi: float = None):
```

Comment:
Any reason we don't just forward all kwargs?

Reply:
No reason, i'll change